docs/_includes/layouts/pages/tokens.11ty.ts

```diff
@@ -114,170 +114,150 @@ export default class TokensPage extends Renderer<Data> {
-                    <table class="${classes}"
-                      style="${styleMap({ '--samp-color': isColor ? token.$value : 'initial' })}">
-                      <thead>
-                        <tr>
-                          <th scope="col" data-label="Example"><abbr title="Example">Ex.</abbr></th>
-                          <th scope="col" data-label="Token name">Token name</th>
-                          <th scope="col" data-label="Value">Value</th>
-                          <th scope="col" data-label="Use case">Use case</th>
-                          <th scope="col" data-label="Copy"></th>
-                        </tr>
-                      </thead>
-                      <tbody>
-                        <tr id="${token.name}-rgb" style="--color: rgb(${r}, ${g}, ${b})">
-                          <td class="sample">
-                            <samp class="${classes}">${token.path.includes('text') ? 'Aa' : example}</samp>
-                          </td>
-                          <td class="token name">
-                            <uxdot-copy-button>--${token.name}-rgb</uxdot-copy-button>
-                          </td>
-                          <td><code>rgb(${r}, ${g}, ${b})</code></td>
-                          <td>To modify opacity</td>
-                          ${copyCell(token)}
-                        </tr>
-                        <tr id="${token.name}-hsl" style="--color: hsl(${h} ${s}% ${l}%)">
-                          <td class="sample">
-                            <samp class="${classes}">${token.path.includes('text') ? 'Aa' : example}</samp>
-                          </td>
-                          <td class="token name">
-                            <uxdot-copy-button>--${token.name}-hsl</uxdot-copy-button>
-                          </td>
-                          <td><code>hsl(${h} ${s}% ${l}%)</code></td>
-                          <td>To modify opacity</td>
-                          ${copyCell(token)}
-                        </tr>
-                      </tbody>
-                    </table>
-                  </rh-table>
-                </details>
-              </td>
-            </tr>
-            `}`;
-            }))).join('\n')}
+            ${(await Promise.all(tokens.filter(x => !x.$deprecated).map(token => this.#renderToken(token, options)))).join('\n')}
           </tbody>
         </table>
-      </rh-table>`.trim();
+      </rh-table>${!deprecatedTokens.length ? '' : html`
+
+      <rh-disclosure summary="Deprecated tokens"
+                     style="--rh-color-surface: light-dark(var(--rh-color-surface-lighter), var(--rh-color-surface-darkest))">
+        <rh-table>
+          <table>
+            <thead>
+              <tr>
+                <th scope="col"><abbr title="Example">Ex.</abbr></th>
+                <th scope="col">Token name</th>
+                <th scope="col">Value</th>
+                <th scope="col">Deprecation reason</th>
+                <th scope="col"></th>
```

Comment:
```suggestion
                <th scope="col">Action</th>
```

This can be visually hidden if needed.